### ISSUE TITLE
Signature help shows up even after completing the call

### ISSUE THREAD

**Issue by dhruvmanila** [astral-sh/ty#879]
### Summary

Playground code:

```py
def foo(x: int, y: str) -> None: ...

foo(1, "")
```

The signature help shows up after _moving_ the cursor after the closing parentheses (`)`) that ends the call. The closing parenthesis was added automatically when I added the opening parenthesis but I explicitly typed <kbd>)</kbd> which _moved_ the cursor beyond the closing parenthesis. And, at this point, the signature help was shown which highlighted the first parameter.

Although, it's interesting that adding the closing parenthesis manually doesn't trigger the signature help. I haven't tried this out in an editor though.

https://github.com/user-attachments/assets/45ae4d17-64b5-43a7-b031-0470ce4806ab

### Version

e0149cd9f

**Comment by UnboundVariable** [comment]
Thanks for reporting. I've checked in a fix.
